Source/9_Resources/BSML/Leaderboard/Components/Header/CaptorClan.bsml

```diff
@@ -0,0 +1,9 @@
+﻿<!-- TODO: How do you make the clanTag smaller than the pref-height of the container it's in?-->
+<horizontal active="~captor-clan-active" pad-left="2" pad-right="2" bg="round-rect-panel" bg-color="#FFFFFFDD" apply-on-change="true" hover-hint="~captor-clan-hover">
+    <horizontal pad-right="1">
+        <text text="~captor-clan-text" color="~captor-clan-text-color" align="Midline"/>
+    </horizontal>
+    <vertical active="~leaderboard-captured" id="background" vertical-fit="PreferredSize" pref-height="5" bg="round-rect-panel">

```

Comment:
instead of cloning ClanTag logic, use it

Reply:
Fixed; We now use logic from clanTag.cs for the clan tag